zeroex/orderwatch/order_watcher.go

```diff
@@ -1093,7 +1355,7 @@ func (w *Watcher) findOrdersToUnexpire(latestBlockTimestamp time.Time) ([]*types
 			{
 				Field: db.OFFillableTakerAssetAmount,
 				Kind:  db.NotEqual,
-				Value: 0,
+				Value: big.NewInt(0),
```

Comment:
Nice catch! I know this is a bit of a footgun in the `db` package. Eventually we can address it by implementing runtime type checking to make sure that `Value` has the same type as the corresponding `Field`.